packages/wallets/src/components/WalletCardIcon/WalletCardIcon.tsx

```diff
@@ -43,21 +42,19 @@ const typeToWidthMapper = {
 const typesWithRoundedIcon = ['EUR', 'GBP', 'USD'];
 
 type TProps = {
+    device?: 'desktop' | 'mobile';
     size?: Extract<TGenericSizes, 'lg' | 'md' | 'sm' | 'xl'>;
     type: Omit<string, keyof typeof typeToIconMapper> | keyof typeof typeToIconMapper;
 };
 
-const WalletCardIcon: React.FC<TProps> = ({ size = 'lg', type }) => {
-    const { isMobile } = useDevice();
-
+const WalletCardIcon: React.FC<TProps> = ({ device = 'desktop', size = 'lg', type }) => {
```

Comment:
We can handle `isMobile ? 'mobile' : 'desktop'` here itself right ? So that we dont need to pass in these in the components. We can override this also if we want. 🤔